### PR TITLE
doc for modal and button

### PR DESCRIPTION
**Pull request by jimmymadrigal**

### CODE REVIEW

**Review by jeanpaniagua — APPROVED** [review]

File: packages/modal/src/stories/ModalCustom.bs.stories.ts [hunk 1]
```diff
@@ -27,34 +38,35 @@ export default {
...
-    }
+type StoryModel = UiModalModel & {
+  titleSlot: string
+  bodySlot: string
+  footerSlot: string
+}
 
 const Template = (args: StoryModel) => ({
-  components: { UiModal },
+  components: { UiModal, UiCodeHighlight },
```
Comment (alejandronavarroGL) [inline packages/modal/src/stories/ModalCustom.bs.stories.ts:48]: Might I would be wrong, but you need to add code highlight in the template, don't you?

**Review by alejandronavarroGL — CHANGES_REQUESTED** [review]

File: packages/button/src/docs/Button.doc.mdx [hunk 2]
```diff
@@ -65,6 +65,17 @@ A button can be of anchor tag type `<a>` if stated
...
 </Canvas>
 
+## Props
+
+| Name | Type  | Description |
+| ---- | ---- | ----------- |
+| `anchor` | `boolean` | if true the button is rendered as an anchor tag `<a>` |
+| `toggle` | `boolean` | if true the button shows as state of active or inactive and sends the `toggle` event |
+| `outline` | `boolean` | outline class is added to the button |
+| `variant` | `string` | varian class defined in the theme |
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/button/src/docs/Button.doc.mdx]: Done

File: packages/button/src/stories/Button.bs.stories.ts [hunk 3]
```diff
@@ -353,21 +354,8 @@ Toggle.args = {
...
-  anchor: true,
-}
-Anchor.parameters = {
-  docs: {
-    source: {
-      code: '<ui-button variant="primary" :anchor="true">Anchor</ui-button>',
-    },
-  },
-}
+}
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/button/src/stories/Button.bs.stories.ts]: Done

File: packages/modal/src/docs/Modal.doc.mdx [hunk 4]
```diff
@@ -1 +1,164 @@
-# Modal
\ No newline at end of file
+import { Story, Canvas, Preview, Source } from '@storybook/addon-docs'
```
[the remark this answers is not in the document]
Comment (jimmymadrigal) [reply packages/modal/src/docs/Modal.doc.mdx]: Done

File: packages/modal/src/stories/ModalCustom.bs.stories.ts (hunk 1 above)
Comment (jimmymadrigal) [reply packages/modal/src/stories/ModalCustom.bs.stories.ts]: Done

**Review by alejandronavarroGL — APPROVED** [review]

**Pull request merged**